pkg/controller/sidecar/sync.go

```diff
@@ -16,15 +16,13 @@ import (
 	"k8s.io/klog/v2"
 )
 
-const requeueWaitDuration = 5 * time.Second
+const (
+	requeueWaitDuration = 5 * time.Second
+	localhost           = "127.0.0.1"
+)
 
 func (c *Controller) getScyllaClient() (*scyllaclient.Client, error) {
-	secret, err := c.secretLister.Secrets(c.namespace).Get(c.secretName)
-	if err != nil {
-		return nil, fmt.Errorf("can't get manager agent auth secret %s/%s: %w", c.namespace, c.secretName, err)
-	}
-
-	return controllerhelpers.NewScyllaClientFromSecret(secret, []string{c.hostAddr})
+	return controllerhelpers.NewLocalhostScyllaClient()
```

Comment:
would it be better to inline the function? it no longer needs any value from the object

Reply:
done